scripts/cli/cli_helper.py

```diff
@@ -41,13 +41,20 @@ def format_source(source: List[str]) -> List[TileFiles]:
             )
             return source_json
         except json.JSONDecodeError as e:
-            get_log().debug("Decoding Json Failed", msg=e)
-
-    local_tile_file: List[TileFiles] = []
-    for s in source:
-        local_tile_file.append(TileFiles(output=os.path.splitext(get_file_name_from_path(s))[0], input=[s]))
-
-    return local_tile_file
+            raise Exception(e) from e
+    else:
+        source_dev: List[TileFiles] = []
+        for s in source:
+            gdalinfo: GdalInfo = gdal_info(s, False)
+            if scale > 0:
```

Comment:
What happen to `origin` if `not scale > 0`?

Reply:
Yeah, nothing - I'll refactor this.